javascript/javascript1/week4/Ali-Nematollahi-Homework/get-count.js

```diff
@@ -0,0 +1,10 @@
+function getCount(str) {
+  let counter = 0;
+  const vowels = ["a", "e", "i", "o", "u"];
+  for (i in str) {
```

Comment:
```suggestion
  for (let i in str) {
```
Don't forget to declare your variables using `let` or `const`